### DISCUSSION TITLE
Limit of Starred

### DISCUSSION THREAD

**Discussion by ramazansancar** [community/community#44191] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Why is there a limit of 32 for lists created in ⭐starred repos?
![image](https://user-images.githubusercontent.com/8878055/212261385-aa2c697c-2f26-4974-8e97-be36e813ec87.png)

I'm going to need more listings to be able to organize my 🌟900+ starred repo. Will this be improved?

Thanks :)

**Comment by radar-blip** [comment]
Not sure why, but at first it was limited to 16. Then they doubled it to 32.
Maybe they will increase it again sometime soon.

**Reply by ramazansancar** [reply]
I hope :)

**Comment by Hlloolee** [comment]
Hloolee

**Reply by Hlloolee** [reply]
Hlloolee

**Reply by Hlloolee** [reply]
Ggf

**Comment by relliv** [comment]
2024 and still list limit is 32. I think it is the right time to increase the limits. 128 might be enough for a while :)

**Comment by ramazansancar** [comment]
I think this limit is insufficient to create an organized list.

![image](https://github.com/user-attachments/assets/bbf5362f-de8c-4ad4-9fa6-09010101174e)
